main.cpp

```diff
@@ -1010,13 +730,373 @@ Part 1e - Step 19: Request a review
+    //     3) Age (int)
+    int age = 4;
+    //     4) Number of eyes (int)
+    int numEyes = 2;
+    //     5) Breed (std::string)
+    std::string breed = "Golden Retriever";
+    // 3 things it can do:
+    //     1) Bark
+    void bark();
+    //     2) Wag tail
+    void wagTail(std::string leftOrRight);
+    //     3) Perform a trick
+    void peformTrick (std::string trickType);
+
+};
+struct Guitar
+{
+
+    // 5 properties:
+    //     1) Number of strings (int)
+    int numStrings = 6;
+    //     2) Number of frets (int)
+    int numFrets = 22;
+    //     3) Volume level (float)
+    float volumeLevel = 3.7f;
+    //     4) Brand (std::string)
+    std::string brand = "Gibson";
+    //     5) Material (std::string)
+    std::string material = "Spruce Wood";
+
+    struct Strings
+    {
+        // Low E string gauge
+        float lowEStringGauge = .040f; 
+
+        float dStringGauge = .010f;
+        std::string stringBrand = "Ernie Ball";
+        std::string sku = "2229";
+        std::string stringMaterial ="Nickel";
+        
+        void stringBend(std::string whichString);
+        void stringSlide(std::string targetNote);
+        void stringSnap(std::string whichString);
+    
+    };
+    // 3 things it can do:
+    //     1) Play individual notes
+    void playNote(std::string whichNote);
+    //     2) Be tuned to different tunings
+    void tuneGuitar(std::string whichString);
+    //     3) Strum notes
+    void strumNotes (std::string whichChord);
+
+};
+struct Wings
+{
+    // 5 properties:
+    //     1) Wing shape (std::string)
+    std::string wingShape = "Airfoil";
+    //     2) Wing length (int)
+    int wingLength = 130;
+    //     3) Ratio of wings length to width (int)
+    int ratioLengthToWidth = 55;
+    //     4) Angle of wing tilt (float)
+    float tiltAngle = 20.f;
+    //     5) Flap angle (float)
+    float flapAngle = 30.f;
+    // 3 things it can do:
+    //     1) Generate lift
+    void generateLift();
+    //     2) Provide stability
+    void stabilizeWings();
+    //     3) Adjust to various conditions and scenarios
+    void emergencyLanding(std::string nearestAirport);
+}; 
+struct Engines 
+{
+//     5 properties:
+//         1) Number of engines (int)
+    int numEngines = 2;
+//         2) Power output (float)
+    float powerOutput = 30.f;
+//         3) Fuel efficiency (float)
+    float fuelEfficiency = 10.f;
+//         4) Emissions (float)
+    float engineEmissions = 20.f;
+//         5) Size (int)
+    int engineSize = 20;
+
+    struct Turbines
+    {
+        int lowPressureTurbineLenth = 84; 
+        int highPressureTurbineWidth = 93;
+        std::string turbineBrand = "GE";
+        std::string turbineModel = "CF3942";
+        std::string turbineMaterial ="Titanium";
+
+        void turbineThrust(std::string whichTurbine);
+        void turbineEfficiency(std::string targetSpeed);
+        void turbineNoise(std::string whichTurbine);
+    };
```

Comment:
Don't include the name of the class in the names of the member variables and member functions.
That's redundant
- [ ] revise